ui/tests/.eslintrc.js

```diff
@@ -29,9 +29,6 @@ module.exports = {
     {
       files: ['integration/components/**/*-test.js'],
       plugins: ['ember-a11y-testing'],
-      rules: {
-        'ember-a11y-testing/a11y-audit-called': 'error',
-      },
```

Comment:
I think we want to keep this, and only disable in specific cases.

Reply:
Hey, great callout. Upon examining this package in our `package.json` this package is pointing to a commit in a repo and not the npm registry. No other HashiCorp product has this pattern. Its worth creating a follow-up issue to edit this behavior and be in line with HashiCorp's a11y standards.

Currently, this rule creates a lot of noise in our tests.